src/Components/Trainers/index.js

```diff
@@ -1,11 +1,57 @@
 import styles from './trainers.module.css';
+import Table from './Table';
+/* import Modal from './Modal'; */
+import { useEffect, useState } from 'react';
+
+const Trainers = () => {
+  const [trainers, setTrainers] = useState([]);
+  const [showDeleteModal, setShowDeleteModal] = useState(false);
+  const [deleteModalMessage, setDeleteModalMessage] = useState('');
+
+  const getTrainers = async () => {
+    const response = await fetch('https://nai-megarocket-server.vercel.app/api/trainers');
```

Comment:
it is recommended to use the variable from the server's .env file
```
      const response = await fetch(`${process.env.REACT_APP_API_URL + '/trainers'}`);

```

Reply:
Thank you for your review. The changes have been completed and it's ready!